### PR TITLE
WFCORE-671 - update tests for RFE ConfigurationChangeNotifications

### PR DESCRIPTION
**Pull request by hsvabek**
update Configuration change notification tests - add some read operations, write operation with FAILED status, invalid write operation
https://issues.jboss.org/browse/WFCORE-671

### CODE REVIEW

File: testsuite/rbac/src/test/java/org/jboss/as/test/integration/mgmt/access/ConfigurationChangesHistoryTestCase.java [hunk 1]
```diff
@@ -50,6 +50,7 @@
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.RESULT;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SENSITIVITY_CLASSIFICATION;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SUCCESS;
+import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.FAILED;;
```
Comment (jamezp) [inline testsuite/rbac/src/test/java/org/jboss/as/test/integration/mgmt/access/ConfigurationChangesHistoryTestCase.java]: The double `;;` needs to be removed.

**Comment by jamezp** [comment]
This is okay once the fix for the double `;;` has been done.

**Comment by bstansberry** [comment]
Please fix the commit message; EAP7-89 is invisible to many people who use this codebase. Use WFCORE.

**Comment by hsvabek** [comment]
retest this please

**Comment by hsvabek** [comment]
retest this please

**Pull request merged**